src/main/java/com/powsybl/openloadflow/network/impl/LfTieLineBranch.java

```diff
@@ -120,11 +120,11 @@ public List<LfLimit> getLimits1(final LimitType type, LimitReductionManager limi
     public List<LfLimit> getLimits2(final LimitType type, LimitReductionManager limitReductionManager) {
         switch (type) {
             case ACTIVE_POWER:
-                return getLimits2(type, getHalf2().getActivePowerLimits().orElse(null), limitReductionManager);
-            case APPARENT_POWER:
```

Comment:
`CURRENT` and `APPARENT_POWER` cases have been swapped here.

Reply:
Reverted